azext_edge/edge/providers/orchestration/resources/dataflows.py

```diff
@@ -46,10 +59,14 @@ def list(self, instance_name: str, resource_group_name: str) -> Iterable[dict]:
         return self.ops.list_by_resource_group(resource_group_name=resource_group_name, instance_name=instance_name)
 
 
-class DataFlows:
-    def __init__(self, ops: "DataflowOperations", get_ext_loc: GetInstanceExtLoc):
-        self.ops = ops
-        self.get_ext_loc = get_ext_loc
+class DataFlows(Queryable):
+    def __init__(self, cmd):
+        super().__init__(cmd=cmd)
+        self.iotops_mgmt_client = get_iotops_mgmt_client(
+            subscription_id=self.default_subscription_id,
+        )
+        self.ops: "DataflowOperations" = self.iotops_mgmt_client.dataflow
+        self.instances = Instances(self.cmd)
```

Comment:
Why is `DataFlows` changed to init a new `Instances` when it only uses show? And when `DataFlows` is used through a nested `DataFlowProfiles` reference (which makes sense), what is the need to initialize a new `iotops_mgmt_client`?